features-guide/index.rst

```diff
@@ -1,5 +1,68 @@
-QWAT features
-=============
+QWAT / TEKSI Drinking Water
+===========================
 
-.. raw:: html
-   :file: QWAT_Feature_grid.html
+QWAT is a PostgreSQL - PostGIS data model almost compatible with the swiss norm SIA405. The data are accessed and edited with a dedicated QGIS project.
```

Comment:
```suggestion
The TEKSI drinking water module (project name: QWAT) is a decision support tool that allows you to:

* manage and map the data of your drinking water network with all its components such as pipes, valves, pumps, tanks, etc.
* indicate the characteristics of network objects in the form of attributes such as diameter, material, installation depth, leaks, date of installation, etc.
* produce plans and extract statistics from the database such as network value, total length of pipes, identification of future interventions, etc.
* export geodata in compliance with the requirements of Swiss standards

```

@tproduit  it would be a great thing to include website introduction to the module!

Reply:
What do you mean with "website introduction"?

Reply:
A similar introduction which is provided on www.teksi.ch but extended here.